SystemInformer/syssccpu.c

```diff
@@ -1078,17 +1079,38 @@ VOID PhSipNotifyCpuGraph(
                             format[14].u.String.Length = cpuType->Length;
                             format[14].u.String.Buffer = cpuType->Buffer;
                             PhInitFormatS(&format[15], L"\n");
-                            PhInitFormatSR(&format[16], PH_AUTO_T(PH_STRING, PhGetStatisticsTimeString(NULL, getTooltipText->Index))->sr);
 
-                            PhMoveReference(&CpusGraphState[Index].TooltipText, PhFormat(format, RTL_NUMBER_OF(format), 0));
+                            if (PhIsCoreParked(Index))
+                            {
+                                PhInitFormatS(&format[16], L"Parked\n");
+                                timeStatisticsPosition = 17;
+                            }
+                            else
+                            {
+                                timeStatisticsPosition = 16;
+                            }
+                            
                         }
                         else
                         {
                             PhInitFormatS(&format[13], L"\n");
-                            PhInitFormatSR(&format[14], PH_AUTO_T(PH_STRING, PhGetStatisticsTimeString(NULL, getTooltipText->Index))->sr);
 
-                            PhMoveReference(&CpusGraphState[Index].TooltipText, PhFormat(format, 15, 0));
+                            if (PhIsCoreParked(Index))
```

Comment:
Tooltips can be fussy. Might need to try to cache this information. Might be an opportunity to eliminate an allocation. Agreed the formatting indexes could do some cleanup.

Reply:
Unlike the other parameters, the parking state of a set can be changed quite often by the processor power manager, so I'm not sure about caching. But I think it'd be worth at least adding a delay of half a second or a bit more.